Tools/autotest/common.py

```diff
@@ -1323,6 +1323,28 @@ def wait_servo_channel_value(self, channel, value, timeout=2):
             if m_value == value:
                 return
 
+    def wait_servo_channel_exceed_value(self, channel, value, timeout=2):
```

Comment:
This is fine, but if we merge it as-is I might come back later and add an optional `operator=operator.eq` keyword parameter to `wait_servo_channel_value`, based on https://stackoverflow.com/questions/18591778/how-to-pass-an-operator-to-a-python-function

The call to `self.wait_servo_channel_exceed_value(5, 1100)` would become `self.wait_servo_channel_value(5, 1100, operator=operator.gt)`

Reply:
I'd also prefer a more elegant approach here